common/SmokeTests/SmokeTest/SmokeTest.csproj

```diff
@@ -13,7 +13,7 @@
     <PackageReference Include="Azure.Core" Version="1.8.0" />
     <PackageReference Include="Azure.Identity" Version="1.4.0-beta.1" />
     <!-- The OverrideDailyVersion attribute prevents the Update-Dependencies script from overwriting it with a daily build version -->
-    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.3.0-beta.4" OverrideDailyVersion="" />
+    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.3.0" OverrideDailyVersion="" />
     <PackageReference Include="Azure.Messaging.EventHubs.Processor" Version="5.3.0-beta.4" />
```

Comment:
You may want to push this to 5.3.0 GA as well to keep it in sync with the main package.

Reply:
Updated. It ends up getting overridden in the build anyways by the alpha version, but since Azure.Messaging.EventHubs is pinned, that's why it had an error.